corehq/apps/users/dbaccessors/couch_users.py

```diff
@@ -14,3 +14,11 @@ def get_user_id_by_username(username):
     if row:
         return row["id"]
     return None
+
+
+def get_display_name_for_user_id(domain, user_id, default=None):
```

Comment:
Good eye. It's not obvious from this code, but `CouchUser.get_by_user_id` is already cached, I believe with enough sophistication that it's properly cleared when a user is saved, so caching this function is at best redundant and at worst would lead to the minor stale cache issues.